docs/_pages/contributing.cn.md

```diff
@@ -88,7 +88,11 @@ and more importantly also try to improve the process along the way as well.
 
  - Once you finalize on a project/task, please open a new JIRA or assign an existing one to yourself. 
       - Almost all PRs should be linked to a JIRA. It's always good to have a JIRA upfront to avoid duplicating efforts.
-      - If the changes are minor, then `[MINOR]` prefix can be added to Pull Request title without a JIRA. 
+      - If the changes are minor, then `[MINOR]` prefix can be added to Pull Request title without a JIRA. There are some tips to help to judge **MINOR** Pull Request :
```

Comment:
I guess it would look better with - "Below are some tips to judge **MINOR** Pull Request"?